### ISSUE TITLE
Speed benchmarks vs scipy

### ISSUE THREAD

**Issue by ogencoglu** [cubao/concave_hull#26]
And speed benchmarks vs scipy / opencv / scikit-image?

**Comment by district10** [comment]
No. This is mainly about concave hull. The convex hull implemented here is only for minimal dependency (we used to depend on scipy convex hull).

**Comment by district10** [comment]
There's a bug in convex hull generation in version 0.0.6~0.0.8 (seems only on macOS and windows, I'm not quite sure).

So the https://github.com/cubao/concave_hull/blob/master/src/convex_hull.hpp ([cp-algorithms/grahams-scan](https://cp-algorithms.com/geometry/convex-hull.html#grahams-scan-algorithm) version is replaced by https://github.com/cubao/concave_hull/blob/master/src/convex_hull2.hpp (just like what [mapbox/cancaveman](https://github.com/mapbox/concaveman/pull/17/files) did).
